src/pubsub/ua_pubsub_config.h

```diff
@@ -0,0 +1,118 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Copyright (c) 2020 Yannick Wallerer, Siemens AG
+ * Copyright (c) 2020 Thomas Fischer, Siemens AG
+ */
+
+#ifdef UA_ENABLE_PUBSUB_CONFIG
+
+#ifndef UA_PUBSUB_CONFIG_H_
+#define UA_PUBSUB_CONFIG_H_
+
+#include <open62541/types_generated.h>  /* Defines UA data types */
+#include <open62541/server.h>           /* Defines UA_Server     */
+
+
+/* UA_decodeBinFile() */
+/**
+ *  @brief      Decodes buffered binary config file
+ * 
+ *  @param      buffer      [in]    Pointer to buffer that buffers the bin file
+ *  @param      offset      [bi]    Offset in the buffer
+ *  @param      dst         [out]   Pointer to Object that shall contain the decoded data
+ * 
+ *  @return     UA_STATUSCODE_GOOD on success
+ */
+UA_StatusCode 
+UA_decodeBinFile
+(
+    /*[in]*/    const UA_ByteString *buffer, 
+    /*[bi]*/    size_t offset,
+    /*[out]*/   UA_ExtensionObject *dst
+);
```

Comment:
This function isn't pubsub file configuration specific. Why do we need it within the API?